### PR TITLE
TV Cinema Mode

### PR DESCRIPTION
**Pull request by ryanmccombe**
Requirement for bbc/simorgh#7178

**Overall change:**
Implement Dark Mode for On Demand TV

**Code changes:**

- _Add toggle for TV Dark Mode, and toggle it on in test_
- _Bump & pass props to Psammead Components_
- _Implement and use new Dark Mode helper_

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by pharingee — APPROVED** [review]

File: src/app/lib/utilities/darkMode/index.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { createGlobalStyle } from 'styled-components';
+
+// TODO: #121212 is a new colour not in psammead - add it?
```
Comment (pharingee) [inline src/app/lib/utilities/darkMode/index.jsx]: Yup. Add it

**Comment by rebeccamcginn** [comment]
This is exciting!

**Comment by rebeccamcginn** [comment]
What does this look like for an expired episode and NoJS?

**Comment by ryanmccombe** [comment]
> What does this look like for an expired episode and NoJS?

Screen Shot 2020-07-10 at 14 56 05

**Review by rebeccamcginn — APPROVED** [review]
Woop woop

**Review by karinathomasbbc — COMMENTED** [review]
👍 Looks good!

**Review by karinathomasbbc — APPROVED** [review]

**Pull request merged**